scripts/cluster/start.sh

```diff
@@ -57,13 +57,15 @@ if [ -z "${GSA_EMAIL}" ]; then
   done
 fi
 
-if ! kubectl get secret/gcsfuse-secrets --namespace kube-system; then
-  KEY_PATH=$(mktemp -d)/key.json
-  gcloud iam service-accounts keys create "${KEY_PATH}" --iam-account "${GSA_EMAIL}"
-  kubectl create secret generic gcsfuse-secrets --from-file="${KEY_PATH}" --namespace kube-system
-  rm -rf $(dirname "${KEY_PATH}")
+KSA_NAME="${GSA_NAME}"
```

Comment:
Could the KSA_NAME be some fixed value like "gcsfuse_sa" so that we don't have to replace it in the yaml?

Reply:
Yes, that's certainly better. Fixed.